app/src/main/java/com/odinwar/mvvmdemoapp/data/model/MovieItem.kt

```diff
@@ -0,0 +1,26 @@
+package com.odinwar.mvvmdemoapp.data.model
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+data class MovieItem(
+    @SerializedName("id")
+    @Expose
+    val id: Int? = null,
+
+    @SerializedName("title")
```

Comment:
tương tự các data còn lại
```suggestion
data class MovieItem(
    @SerializedName("id")
    @Expose
    val id: Int? = null,
    @SerializedName("title")
```